www/js/index.js

```diff
@@ -44,15 +44,15 @@ var app = {
     onIntegrate: function () {
         document.addEventListener("resume", app.onResume, false);
         document.addEventListener("pause", app.onPause, false);
-        Localytics.integrate("YOUR-LOCALYTICS-APP-KEY");
+        Localytics.integrate("dcdf9cb70fff87655848357-8f7e9180-385a-11e4-a3d0-009c5fda0a25");
```

Comment:
Dont like the appKey here. the Old code is right.

Reply:
ya just a mistake after testing - will fix